sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -323,10 +318,18 @@ export class ManagementClient extends LinkEntity {
         options = {};
       }
 
+      const operationTimeoutInMs =
+        options.retryOptions == undefined ||
```

Comment:
This logic is duplicated in `eventHubReceiver`, and will probably be used by the receiver at some point. Could you refactor this into its own function that takes in either `retryOptions` or `operationTimeoutInMs`, and then reference this function in both places?

Reply:
Finding a place to house a utility like that seemed a bit tricky. Updated/created one along side RetryOptions interface definition in `eventHubClient` file.

Moreover, this may change and all required validations would likely be done in one place as part of `retry` module within `core-amqp` eventually.